src/db/migrations/28-add-mighty-33-others-die.ts

```diff
@@ -37,7 +37,8 @@ const OTHERS_DIE: NftFactory = {
       type: MusicPlatformType['multi-track-multiprint-contract'],
       extractor: {
         title: TitleExtractorTypes.METADATA_NAME_WITHOUT_LEADING_INFO,
-        id: IdExtractorTypes.USE_TITLE_EXTRACTOR
+        id: IdExtractorTypes.USE_TITLE_EXTRACTOR,
+        websiteUrl: WebsiteUrlExtractorTypes.METADATA_EXTERNAL_URL
```

Comment:
As you mentioned a while back, we should stop changing past migrations as that doesn't really work unless we rebuild the whole DB. We'd need to do these changes in migration 29 ideally.

(But, I don't think these migrations are necessary anyway given my other comment on default urls)

Reply:
Cool, makes sense